### ISSUE TITLE
Ability to use EM without Register 

### ISSUE THREAD

**Issue by CheloXL** [expressmapper/ExpressMapper#38]
Hi,
Right now I'm using ValueInjector as it simply let me do myObject.InjectFrom(otherObject). Having to register every mapping I have is simply impossible, as I usually have several mappings for different classes that follows some interface definitions.

Is there a way I can simply run something similar and EM do the registration on the fly?

**Comment by anisimovyuriy** [comment]
Hi CHeloXL,

Thanks for your valuable input!
Yes it is possible, but if there'd be no custom mapping members like:
```Mapper.Register<Model, ViewModel>()
.Member(dst => dst.Name, src => string.Format("{0} {1}", src.FirstName, src.LastName));```

If it is just like:
```Mapper.Register<Model, ViewModel>();```
then no problem.
As the main intent for Expressmapper is performance and user friendliness. So you would be able to do the following without registration and without customizing member mappings:
```Mapper.Map<Model, ViewModel>();```
The registration will happen implicitly.
Please let me know if it works for you.

**Comment by CheloXL** [comment]
Well, Map may work, but I already have instantiated my target object. In fact, I usually inject the source object into the target object on the target's ctor.

And yes, we are talking about "normal" member mapping (same name, same type).

**Comment by anisimovyuriy** [comment]
Great. Then you can use either:
```Mapper.Map<Model, ViewModel>(src);```
or
```Mapper.Map<Model, ViewModel>(src, dest);```

Where dest - your target instance and src your source instance.
Will it work?

**Comment by CheloXL** [comment]
Mapper.Map<Model, ViewModel>(src, dest); will do the work... Probably I will create an extension method for that named InjectInto so I don't have to refactor my code. But that makes my life easier for sure as I really don't want to create a registration class for all my mappings (at least not on the code I'm planning to use, that's huge enough to have think about this things two or three times before doing any major refactoring).

Thank you!

**Comment by anisimovyuriy** [comment]
Fixed - will be included in 1.5 release
